src/components/EditListLayout/EditListLayout.tsx

```diff
@@ -10,7 +10,7 @@ type EditListLayoutProps = {
   isLoading?: boolean;
   onCancel?: () => void;
   onSave?: () => void;
-  children?: ReactElement[],
+  children?: ReactNode,
```

Comment:
ReactNode is a superset of ReactElement; not strictly needed for this PR, but makes it easier to debug with, if nothing else.
![image](https://github.com/folio-org/ui-lists/assets/8005215/69e90837-ac94-485a-aebd-6ac8064dcb19)

Reply:
Yikes! Stay away from that last type :D